### ISSUE TITLE
Can this be used as a backup and restore tool?

### ISSUE THREAD

**Issue by Nostradamus1973** [MMMZZZZ/Jellyfin-Migrator#7]
Good evening,

I've been using Jellyfin for a few years and recently lost everything because I tried backing up the /etc/jellyfin and the /var/lib/jellyfin/data folder as suggested by a few forums and wrecked everything several times.  I'm really hoping your script is the sort of thing I've been seeking out  for quite some time now.

So, I wanted to know if I fill out the variables in your script, will I be able to use it as a backup and restore? Or, can it onlly migrate?

Thank you

**Comment by MMMZZZZ** [comment]
This tool is (only) useful when you‘re changing file names or file paths. A simple copy/restore to and from the same location shouldn’t break anything if all your config and media paths stay the same. Maybe you have an issue with file permissions…?

Kind regards,
Max

**Comment by Nostradamus1973** [comment]
Yes, you are 100% correct, I use Deja-Dup and I needed to change the permissions from root to the current user and I was able to back up nicely with it.
Have a great day.
